### PR TITLE
Tabs

### PR DESCRIPTION
**Pull request by djsaun**
Completed the Tabs/Accordion component. The CSS is contained in a postcss mixin that includes an optional min-width breakpoint argument. 

Please let me know if you have any questions or concerns!

### CODE REVIEW

File: src/components/tabs/tabs.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      element.children[0].classList.remove('active');
+      element.children[1].classList.remove('is-current');
+    });
+
+    this.classList.add('active');
+    this.children[0].classList.add('active');
+    this.children[1].classList.add('is-current');
+  };
+
+  for (const tab of tabList) {
```
Comment (defvayne23) [inline src/components/tabs/tabs.js]: Any reason why you're using a `for` here and a `foreach` on line 11.

Comment (defvayne23) [reply src/components/tabs/tabs.js]: It's been decided to replace `forEach` with `for` in favor of Babel compiling down to EC2015.

File: src/components/tabs/tabs.css [hunk 2]
```diff
@@ -0,0 +1,180 @@
+@define-mixin tabs $breakpoint {
+  #tabs {
+    margin: 40px auto;
+    padding: 0 0 15px;
+    float: none;
+    text-align: justify;
+
+    &:after {
```
Comment (defvayne23) [inline src/components/tabs/tabs.css]: This needs to be double colon.

Comment (defvayne23) [reply src/components/tabs/tabs.css]: Apply to other pseudo elements within the document.

**Review by fleeting — APPROVED** [review]

**Review by defvayne23 — APPROVED** [review]
:shipit:

**Pull request merged**